recipes/ganglia_install.rb

```diff
@@ -19,7 +19,7 @@
 when 'MasterServer', nil
   case node['platform']
   when "redhat", "centos", "amazon", "scientific" # ~FC024
-    package %w[ganglia ganglia-gmond ganglia-gmetad ganglia-web httpd php php-gd rrdtool] do
+    package %w[ganglia ganglia-gmond ganglia-gmetad ganglia-web httpd php php-gd php-xml rrdtool] do
```

Comment:
Is this `php-xml ` package name consistent among all the supported OSs under this platform condition?

Reply:
I verified Alinux and Alinux2 and the package is available and I ran kitchen tests for all the OSes